entrypoints/popup/App.tsx

```diff
@@ -35,18 +36,35 @@ const defaultUserDataInfo: UserDataObj = {
   ip: '-',
 }
 
-export const App = ({
-  storedUserData,
-}: {
-  storedUserData: Record<string, unknown>
-}) => {
-  const [userData, setUserData] = useState<UserDataObj>(
-    storedUserData.userData as UserDataObj
-  )
+export const App = () => {
+  const [userData, setUserData] = useState<UserDataObj>({
+    location: '-',
+    ip: '-',
```

Comment:
why '-' instead of null? it is for display?

Reply:
Yep, it is for the UI